vscode-udfs/README.md

```diff
@@ -0,0 +1,93 @@
+<!-- title: How to create UDFs with Confluent for VS Code -->
+<!-- description: In this tutorial, learn how to develop, upload, register and execute Flink UDFs with Confluent for VS Code, with step-by-step instructions. -->
+
+# VSCode Flink UDFs
+
+[Install Confluent for VS Code](https://docs.confluent.io/cloud/current/client-apps/vs-code-extension.html).
```

Comment:
Add a prereqs section that includes:
* Confluent Cloud account
* installing the extension
* Java version 11 or 17 (I got bit by this, happened to have java 8 enabled and it wont compile with 8)
* Maven